openrtb_ext/device.go

```diff
@@ -12,14 +14,58 @@ const PrebidExtKey = "prebid"
 
 // ExtDevice defines the contract for bidrequest.device.ext
 type ExtDevice struct {
+	// Attribute:
```

Comment:
Trying out a new comment format, mimicking what exists in the OpenRTB library. Whatchya think?